contracts/src/token/erc721/mod.rs

```diff
@@ -1141,21 +1141,18 @@ mod tests {
     }
 
     #[motsu::test]
-    fn balance_of_zero_balance(contract: Erc721) {
+    fn error_when_contract_balance_is_zero_balance(contract: Erc721) {
```

Comment:
I can see this test is not checking any error. I think we are not expecting to change its name. cc @bidzyyys

Reply:
Yep, we should rename only when we check errors in tests.